apps/client/src/features/viewers/studio/StudioClock.jsx

```diff
@@ -73,16 +78,15 @@ export default function StudioClock(props) {
     <div className={`studio-clock ${isMirrored ? 'mirror' : ''}`} data-testid='studio-view'>
       <NavigationMenu />
       <div className='clock-container'>
-        <div className='studio-timer'>{clock}</div>
+        <div className={`${showSeconds ? 'studio-timer-seconds' : 'studio-timer'}`}>{clock}</div>
```

Comment:
after fiddeling around with useFitText for about thre hours i've decided to go the CSS class way. You are now able to show the seconds vie the ?seconds=true url parameter